vmdb/app/controllers/ems_common.rb

```diff
@@ -336,17 +340,16 @@ def update_button_validate
     set_record_vars(verify_ems, :validate)
     @in_a_form = true
     @changed = session[:changed]
-    begin
-      result = verify_ems.verify_credentials(params[:type])
-    rescue StandardError => bang
-      add_flash("#{bang}", :error)
+
+    # validate button should say "revalidate" if the form is unchanged
```

Comment:
@h-kataria What do you think of these UI changes?

The problem is if the current status of the creds is invalid and you edit the Ems, the validate button WITH NO CHANGES in the form, should revalidate the creds and UPDATE the status.  As soon as you make changes to the form, the validate button SHOULD NOT update the status and authentication (since that bypasses save).